dev/com.ibm.ws.microprofile.openapi_fat/fat/src/com/ibm/ws/microprofile/openapi/fat/utils/OpenAPIConnection.java

```diff
@@ -136,6 +136,7 @@ public OpenAPI downloadModel() throws Exception {
     public HttpURLConnection getConnection() throws IOException, ProtocolException {
         HttpURLConnection conn = HttpUtils.getHttpConnection(constructUrl(), expectedResponseCode, null, 30, method,
             headers, streamToWrite);
+        conn.setReadTimeout(30 * 1000);
```

Comment:
Are these two "30" seconds supposed to be the same thing? Shoud it be a constant? Should the ReadTimeout be set inside HttpUtils at the same time as the ConnectTimeout?

Reply:
Not necessarily, the connect timeout is how long you wait for the connection to be estabilshed whereas the read timeout is how long you wait for the server to say something after the application tries to read from the connection. For the OpenAPI tests, we always expect a prompt response so it's safe to set a read timeout (and hopefully this will result in a more useful failure than the whole test bucket timing out). The 30 seconds is somewhat arbitrary in both cases, we don't expect the timeout to ever be hit, it's just a failsafe.

I didn't want to change `HttpUtils` since that's a global test class and we have tests which expect the server to take a long time to respond (e.g. `FATServletClient`) so setting a read timeout would not be appropriate.